R/shared-methods.R

```diff
@@ -96,13 +96,13 @@ initialise_dataclass <- function(self, level = "1", regions,
     self$target_regions <- regions
   }
 
-  if (!is.null(self$level_data_urls[[self$level]])) {
+  if (length(self$common_data_urls) > 0) {
```

Comment:
don't we need both the length check for common and level as merge will fail if either are list()?

Reply:
Yeah Im going to make a PR with the change as individually is alowing mistakes to creap in. #257